innopoints/views/notification.py

```diff
@@ -0,0 +1,35 @@
+"""Views related to notifications.
+
+- GET   /notifications
+- PATCH /notifications/{notification_id}/read
+"""
+
+from flask import abort
+from flask_login import login_required, current_user
+
+from innopoints.blueprints import api
+from innopoints.extensions import db
+from innopoints.models import Notification
+from innopoints.schemas import NotificationSchema
+
+
+@api.route('/notifications')
+@login_required
+def get_notifications():
+    """Gets all notifications of the current user."""
+    query = Notification.query.filter_by(recipient_email=current_user.email)
+    notifications = query.all()
+    schema = NotificationSchema(many=True)
+    return schema.jsonify(notifications)
+
+@api.route('/notifications/<int:notification_id>/read', methods=['PATCH'])
```

Comment:
Also, I'd opt for `POST` here, since the endpoint is `read`, it defines the action

Reply:
I was thinking of it as "patching" the notification to edit the "read" (read "red", the past tense form) attribute of the notification

Reply:
Okay, fair enough. Here's a reasonable guideline: https://stackoverflow.com/a/31698882/5726823

But in that case we'll have to redo a few other endpoints from POST to PATCH :) but I'll take care of it eventually too